doc/source/limitation.rst

```diff
@@ -74,3 +74,11 @@ The following limitations apply to |PXC|:
 
      Galera Documentation: Tables without Primary Keys
         http://galeracluster.com/documentation-webpages/limitations.html#tables-without-primary-keys
+
+* As of version 5.7.32-13.47, an INPLACE `ALTER TABLE <https://dev.mysql.com/doc/refman/5.7/en/alter-table.html>`__  query takes a shared lock on the table throughout the duration of the query. 
+
+  This change addresses a deadlock, which could cause a cluster node to hang in the following scenario:
+
+  * An INPLACE ``ALTER TABLE`` query in one session or being applied as Total Order Isolation (TOI) 
+
+  * A DML on the same table from another session
```

Comment:
In addition to this, we should also document that `LOCK=NONE` clause is no longer allowed for all the inplace ALTER TABLE queries for the reason that they now internally take a shared lock during the execution.

The same has been tested in our MTR: https://github.com/percona/percona-xtradb-cluster/pull/1290/files#diff-e54271faf0750632c2bd6529abbc278bef99901a43726165537bb4de0f49f0feR175-R176